azavea/district-builder-dtl-pa
# Add production deployment environment variables

## Overview

Add `PROD_DB_SETTINGS_BUCKET` and `PROD_DB_DOCKER_HOST` to `.travis.yml`, and use them to override the defaults in `docker-compose.ci.yml` when running production deploys.

### Checklist

- [ x ] PR has a name that won't get you publicly shamed for vagueness
~- [ ] Files changed in the PR have been `yapf`-ed for style violations~

### Demo

https://pa.districtbuilder.azavea.com

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.


## Testing Instructions

 * This was deployed to Production in this Travis build: https://travis-ci.org/azavea/district-builder-dtl-pa/builds/424906413

* See https://pa.districtbuilder.azavea.com

Connects #83
